src/index.tsx

```diff
@@ -10,7 +10,11 @@ import { reportWebVitals } from 'lib/reportWebVitals'
 
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 
-const root = createRoot(document.getElementById('root')!)
+const rootElement = document.getElementById('root')!
+const root = createRoot(rootElement)
+
+rootElement.setAttribute('vaul-drawer-wrapper', '')
+rootElement.classList.add('app-height')
```

Comment:
Do we have to set this here? Feels like a file we shouldn't touch if we can avoid

Reply:
Its supposed to be set on the top level div wrapping the app. But I can prob wrap the entire app in another div, but will have to do a bunch of testing again to see if it still works as expected.

Reply:
@gomesalexandre should I move this off the #root component? Or can we just leave as it, i'm worried if I move it to another div it might screw something up.